packages/plugins/test/mocha/mock.ts

```diff
@@ -69,36 +65,32 @@ export const makeMockSphinxContext = (
       _initCodeWithArgsArray: Array<string>
     ) => {
       const configArtifacts: ConfigArtifacts = {}
-      const files = readdirSync(`./out/artifacts/build-info/`)
-      if (!buildInfos) {
-        buildInfos = await Promise.all(
-          files.map((file) =>
-            JSON.parse(
-              readFileSync(`./out/artifacts/build-info/${file}`, 'utf8')
-            )
-          )
-        )
-      }
```

Comment:
We don't have to do this anymore since we don't recompile during execution, how wonderful.